SaltSquareIO/sports-app-react-fe
# Login page UI

[CLICKUP_TICKET_ID](https://app.clickup.com/t/86c0g7kaz)

## Summary of change

This PR implements a login form with the following elements and functionality:
- An email input field.
- A password input field where the input text is hidden.
- A button to toggle the password visibility (revel/hide password).
- A "Login" button that logs the email and password in the console upon clicking.

## QA Notes

1. Enter an email address in the email input field.
2. Enter a password in the password input field. The password characters should be hidden.
3. Toggle the password visibility using the eye icon button. It should revel/hide the password text.
4. Click the "Login" button. The email and password values should be printed out in the browser console.
